pkg/kubelet/volumes.go

```diff
@@ -77,12 +77,18 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.BoundPod) (volumeMap, error) {
 	for i := range pod.Spec.Volumes {
 		volSpec := &pod.Spec.Volumes[i]
 
+		podRef, err := api.GetReference(pod)
```

Comment:
I think this is going to fail in the masterless case (e.g. ContainerVM).

@lavalamp what are we going to do about this?  Is this an inappropriate use of ObjectRef in your view?  What should we be passing around inside Kubelet to carry namespace, name, and UID?